abcclassroom/quickstart.py

```diff
@@ -0,0 +1,105 @@
+"""
+abc-classroom.quickstart
+========================
+
+"""
+
+import os
+from shutil import copy, rmtree
+from argparse import ArgumentParser
+from abcclassroom import __file__
+
+
+def path_to_example(dataset):
+    """
+    Construct a file path to an example dataset.
+    This file defines helper functions to access data files in this directory,
+    to support examples. Adapted from the PySAL package.
+
+    Parameters
+    ----------
+    dataset: string
+        Name of a dataset to access (e.g., "config.yml")
+            
+    Returns
+    -------
+    string
+        A file path (string) to the dataset
+    """
+    abcclassroom_path = os.path.split(__file__)[0]
+    data_dir = os.path.join(abcclassroom_path, "example-data")
+    data_files = os.listdir(data_dir)
+    if dataset not in data_files:
+        raise KeyError(dataset + " not found in abc-classroom example data.")
+    return os.path.join(data_dir, dataset)
+
+
+def create_dir_struct():
+    """
+    Create a directory structure that can be used to start an abc-classroom course. This includes a main directory,
+    two sub directories for templates and cloned files, and a start to a configuration file.
+    """
+    # Creating the default names for the cloned and template repos.
+    course = "course-dir"
+    # Making sure the configuration file is where it's supposed to be.
+    config = path_to_example("config.yml")
+    if not os.path.exists(config):
+        raise ValueError(
+            "config.yml configuration file can't be located, please ensure abc-classroom has been installed correctly"
+        )
+    # Allows users to rename the cloned and template repos.
+    parser = ArgumentParser(description=create_dir_struct.__doc__)
```

Comment:
We should be consistent across the console scripts about where we define the command line arguments. For all of the other scripts, we do this in __main__.